Hamburg/Stabi/SHOr271a.xml

```diff
@@ -1,316 +1,342 @@
-                                  Indeed, the pricks are more marked in the internal bifolio than in the outer.
-                                </ab>
-                              </layout>
-                              <layout columns="2" writtenLines="26 27">
-                                <note >Data on dimension are taken from <locus target="#68r"/>.</note>
-                                <locus from="64r" to="69v"/>
-                                <dimensions unit="mm" >
-                                  <height>145</height>
-                                  <width>105</width>
-                                </dimensions>
-                                <dimensions type="margin">
-                                  <dim type="top">15</dim>
-                                  <dim type="bottom">34</dim>
-                                  <dim type="right">22</dim>
-                                  <dim type="left">10</dim>
-                                  <dim type="intercolumn">10</dim>
-                                </dimensions>
-                                <ab type="ruling" subtype="pattern">1-1-11/0/0/C</ab>
-                                <ab type="pricking" subtype="pattern">
-                                  The layout corresponds to the pattern I defined by D. Nosnitsin: the text is written above the first ruled line, the vertical and horizontal pricks are visible
-                                  and they are located in the margins.
-                                  Probably, the pricking was made opening the quire in the middle and piercing the folios together.
-                                  Indeed, the pricks are more marked in the internal bifolio than in the outer.
-                                </ab>
-                              </layout>
-                            </layoutDesc>
-                        </objectDesc>
+              <layoutDesc>
+                <layout columns="2" writtenLines="24">
+                  <note>Data on dimension are taken from <locus target="#38r"/>.</note>
+                  <locus from="2r" to="63v"/>
+                  <dimensions unit="mm">
+                    <height>137</height>
+                    <width>107</width>
+                  </dimensions>
+                  <dimensions type="margin">
+                    <dim type="top">21</dim>
+                    <dim type="bottom">38</dim>
+                    <dim type="right">20</dim>
+                    <dim type="left">12</dim>
+                    <dim type="intercolumn">10</dim>
+                  </dimensions>
+                  <ab type="ruling" subtype="pattern">1-1-11/0/0/C</ab>
+                  <ab type="pricking" subtype="pattern"> The layout corresponds to the pattern I
+                    defined by D. Nosnitsin: the text is written above the first ruled line, the
```

Comment:
Maybe better to refer to Denis's publication than just to his name?

Reply:
"pattern I defined" is confusing, because "I" can be read as "me". Better: "The layout corresponds to the pattern 1, defined by D. Nosnitsin."

Reply:
But if Denis uses Latin numbers, I think it's better to keep them. However, maybe "Pattern I", with a capitalized "p", would make it clear? It could also be italicized.

Reply:
I see, that a comma in this place is grammatically incorrect. It is better to write this way:
The layout corresponds to pattern I as defined by D. Nosnitsin: